python/ray/worker.py

```diff
@@ -1231,6 +1236,11 @@ def connect(node,
     # will use glog, which is intialized in `CoreWorker`.
     ray.state.state._initialize_global_state(
         node.redis_address, redis_password=node.redis_password)
+    if mode == SCRIPT_MODE:
+        ray_pkg.driver_runtime_init(job_config)
+    elif mode == WORKER_MODE:
+        ray_pkg.worker_runtime_init(
+            worker.core_worker.get_job_config().runtime_env)
```

Comment:
I commented above with concrete suggestions, but in general consider concrete names like "ensure runtime env setup" rather than "init runtime".